### PR TITLE
fix bug when ranking images with CLIP on inference notebook

### PR DESCRIPTION
**Pull request by bzantium**
hi @borisdayma, thanks for providing great repo! When I used inference notebook, I found there's minor bug when ranking images by CLIP. please check the update! thanks.

- fix bug on inference notebook by reshaping logits

### CODE REVIEW

**Comment by borisdayma** [comment]
I don't think there is a bug.
Basically the first index of logits correspond to the prompt (when you have multiple).
